changelog/unreleased/kong/fix-key-auth-retain-query-order.yml

```diff
@@ -0,0 +1,3 @@
+message: Fixed *key-auth* to retain order of query arguments when hiding the credentials.
```

Comment:
Should this format start with `**key-auth**: `, for example: `**key-auth**: Fixed ...`?